File: screenshot.js
const screenshot = require('screenshot-desktop');

screenshot().then((img) => {
  const fs = require('fs');
  fs.writeFileSync('/Users/konglingzheng/.openclaw/workspace/screenshot.png', img);
  console.log('Screenshot saved to /Users/konglingzheng/.openclaw/workspace/screenshot.png');
}).catch(err => {
  console.error('Error taking screenshot:', err);
  process.exit(1);
});

File: screenshot.py
import mss
import os

# Ensure output directory exists
output_dir = "/Users/konglingzheng/.openclaw/workspace"
os.makedirs(output_dir, exist_ok=True)

output_path = os.path.join(output_dir, "screenshot.png")

with mss.mss() as sct:
    screenshot = sct.shot(output=output_path)
    print(f"Screenshot saved to {screenshot}")
